go/vt/sqlparser/sql.y

```diff
@@ -2227,10 +2235,10 @@ create_table_prefix:
   }
 
 table_spec:
-  '(' table_column_list ')' table_option_list
+  '(' table_column_list ')' table_option_list partition_options
   {
     $$ = $2
-    $$.Options = $4
+    $$.Options = $4 + $5
```

Comment:
Need a space between these?

Reply:
Adding a space here makes it so that tests that don't have a partition option have a dangling space afterwards, which causes a bunch of test failures.
I could add another member variable `PartitionOptions` to separate the two, and make the `Format()` output look better.
I don't think this matters too much though, since we're not using `Options` anywhere outside of just testing